test/e2e_node/lock_contention_test.go

```diff
@@ -0,0 +1,82 @@
+//go:build linux
+// +build linux
+
+/*
+Copyright 2021 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package e2enode
+
+import (
+	"time"
+
+	"golang.org/x/sys/unix"
+
+	"github.com/onsi/ginkgo"
+	"github.com/onsi/gomega"
+	"k8s.io/kubernetes/test/e2e/framework"
+)
+
+const contentionLockFile = "/var/run/kubelet.lock"
+
+var _ = SIGDescribe("Lock contention [Slow] [Disruptive] [NodeSpecialFeature:LockContention]", func() {
+
```

Comment:
Does this test needs `NodeSpecialFeature`? Also is it really `[Slow]` How long is the test? I suggest to keep it as `[Disruptive][Serial][NodeConformance]`. This test is suppose to run everywhere.

Reply:
+1 to `[Disruptive] [Serial]`

-1 to making this `[NodeConformance]` off the bat without signal

`[Slow]` is anything over 5m: https://github.com/kubernetes/community/blob/master/contributors/devel/sig-testing/e2e-tests.md#kinds-of-tests

Reply:
> `[Slow]` is anything over 5m: https://github.com/kubernetes/community/blob/master/contributors/devel/sig-testing/e2e-tests.md#kinds-of-tests

my exact question. Looking at test it doesn't feel it will run long

Reply:
> -1 to making this `[NodeConformance]` off the bat without signal

this will not run under conformance lane as it is Serial. But I am ok to do it incrementally

Reply:
@ehashman @sergeyevstifeev 

Hello, Sorry for the delay in response.

The following tags `[Disruptive] [NodeSpecialFeature:LockContention]` were used because the test restarts the kubelet and using the NodeSpecialFeature was discussed due to the need of this test should not be picked up by any other test suite.

[Serial] tag is not used because this test adds extra lock file contention flags to the test, which if not added to the other tests in the Serial test suite, makes the Serial test suite fail. It was then discussed in the monthly call that the flags shouldn't be added to other tests in the Serial test suite, hence it was advised to use a tag that will ensure that no other test suite will pick this test.